docs/developer/basic-form-support.md

```diff
@@ -0,0 +1,289 @@
+# Basic Form Support
+
+**NOTE:** This feature is under heavy development. Some of the described below may change in the future.
+
+Since Kubeapps 1.6.0, it's possible to attach a chart with a JSON schema that defines the structure of the `values.yaml` file. This JSON schema is used with two goals:
+
+ - Validate that the given values satisfy the schema defined. In case the submitted values are not valid, the installation or upgrade will fail.
+ - Present the user with a simple (basic) form so the chart is easier to deploy and configure.
+
+The goal of the basic form is to define the most common parameters that users typically modify before deploying a chart (like username and password) so they are able to modify them with a more user-friendly form.
```

Comment:
The form itself isn't defining the most common properties - the schema is, right? So perhaps, "The goal of this feature is to present the user with the most common parameters which are typically modified before deploying a chart (like username and password) in a more user-friendly form."